### DISCUSSION TITLE
Does NestedHV mean nested virtualization?

### DISCUSSION THREAD

**Discussion by mszhanyi** [actions/virtual-environments#5332] [Image configuration]
I find there's a parameter _NestedHV_ in Mac.json
https://github.com/actions/virtual-environments/blob/c2797f37cdef66b0c3e6ea41c8bdec543f2152f9/images/macos/templates/macOS-12.json#L37

Could it be used in Windows Image?

**Comment by miketimofeev** [comment]
@mszhanyi I'm afraid no as it's VmWare specific setting
